### ISSUE TITLE
Usage for datasource option

### ISSUE THREAD

**Issue by h7kanna** [ryantxu/ajax-panel#8]
Hi, very useful plugin.
I am trying to figure out what is the use of datasource option for authentication.
Can you provide any usage examples?

Thanks,

**Comment by ryantxu** [comment]
What datasource are you trying to connect to?  The datasource option really just makes requests to the same base path as regular datasource requests, but gives you the response directly.

**Comment by h7kanna** [comment]
@ryantxu 
Hi, I figured out the usage. But thought if the list of examples has one with default 'testdata' datasource as an example. I can create a PR to add one.

Thanks for this plugin.

**Comment by ryantxu** [comment]
yes -- can you make a PR with example usage?  As you can see there are LOTS of options!
